### ISSUE TITLE
Nesting Analysis

### ISSUE THREAD

**Issue by hgn** [terryyin/lizard#202]
Is it possible to add an additional feature to count idents? You desribe it itself in https://github.com/terryyin/lizard/blob/master/theory.rst and is probbaly a nice additional metric for software.

**Comment by terryyin** [comment]
Yes, but it will perhaps take some time. I will have some time in the coming weeks.

**Comment by hgn** [comment]
Great! I am really surprised how rock solid lizard runs, really a great piece of software. Thank you!
